### PR TITLE
URL uniformisation for namespaces prefix (/@ vs /&40) (#601)

### PR DESCRIPTION
**Pull request by fel-mazo**
Hello,

I made the changes described in the ticket.
Let me know if there is something I can improve.

attached, screenshots of what changed

image
image
image
image

### CODE REVIEW

**Comment by tchoutri** [comment]
@fel-mazo Thank you very much! I think the `mkURL` function may also cause problems: https://github.com/flora-pm/flora-server/blob/development/src/web/FloraWeb/Components/PaginationNav.hs#L44-L54

**Comment by fel-mazo** [comment]
yes defo ! will look into it asap

**Comment by fel-mazo** [comment]
I just updated my branch, I also tried to reorganize my changes a bit and put everything in the Links module, let me know if the names are fine as they are or if it would be better to change them.

File: src/web/FloraWeb/Pages/Templates/Packages.hs [hunk 1]
```diff
@@ -70,6 +65,10 @@ import FloraWeb.Components.Utils
 import FloraWeb.Links qualified as Links
 import FloraWeb.Pages.Templates (FloraHTML, TemplateEnv (..))
 import FloraWeb.Pages.Templates.Haddock (renderHaddock)
+import Lucid
+import Servant (ToHttpApiData (..))
+import Text.PrettyPrint (Doc, hcat, render)
+import Text.PrettyPrint qualified as PP
```
Comment (tchoutri) [inline src/web/FloraWeb/Pages/Templates/Packages.hs:71]: Let's keep third-party import statements in the upper paragraph. :)

**Review by tchoutri — CHANGES_REQUESTED** [review]

**Pull request merged**